jsonschema-core.xml

```diff
@@ -135,13 +135,67 @@
-                instances with additional information.  Additional keywords are used to apply
-                assertions and annotations to more complex JSON data structures, or based on
-                some sort of condition.
+                A JSON Schema document is comprised of a set of keywords,
+                which are specified as properties in a JSON object.
+                The name of the keyword is used as the property name,
+                and any arguments to the keyword are provided as the value.
+                Keywords can be classified by their functions.
+                The most important classes are assertion, annotation, and applicator keywords.
             </t>
             <t>
-                To facilitate re-use, keywords can be organized into vocabularies. A vocabulary
+                "Assertion keywords" form the basis of JSON Schema.
+                Given a schema and an instance, the instance is "valid" if all of the keywords in
+                the schema are valid against the instance.
+                Assertion keywords may be used to describe a set of JSON documents.
+                Without any assertion keywords, the set of all valid instances is the set of all valid JSON documents.
+                Typically, adding assertion keywords reduces the set of valid JSON documents.
+                Keywords are defined to not be redundant with other keywords:
+                For example, the "minimum" keyword only shrinks the valid set of numbers, but not strings or other
+                types&#x2014;this is what the "type" keyword does.
+            </t>
+            <t>
+                When an instance validates against a schema, "annotation keywords" may provide "annotations":
+                metadata that describes the instance.
+                For example, you can document the meaning of a property,
+                suggest a default value for new instances,
+                generate a list of hyperlinks from the instance,
+                or declare relationships between data.
+            </t>
+            <t>
+                "Applicator keywords" are keywords that accept sub-schemas as part of their argument,
+                which are validated against the instance in some fashion.
+                Applicator keywords may use or modify the validation result of their sub-schemas,
+                for example, to be valid conditional on which sub-schemas are valid
+                (e.g. the "if", "then", "oneOf", or "anyOf" keywords).
+                They may also validate a schema against values within the instance,
+                for example, a specific property within the object (the "properties" keyword).
+                The "$ref" keyword is a special applicator keyword that takes a reference to a schema,
+                instead of a literal schema. This allows schemas to be defined recursively.
```

Comment:
it also allows schemas to be broken into manageable bits and promote code reuse.